crates/node_binding/binding.d.ts

```diff
@@ -1305,6 +1266,40 @@ export interface JsTap {
   stage: number
 }
 
+export interface KnownAssetInfo {
+  /** if the asset can be long term cached forever (contains a hash) */
+  immutable?: boolean
+  /** whether the asset is minimized */
+  minimized?: boolean
+  /** the value(s) of the full hash used for this asset */
+  fullhash?: string | Array<string>
+  /** the value(s) of the chunk hash used for this asset */
+  chunkhash?: string | Array<string>
+  /**
+   * the value(s) of the module hash used for this asset
+   * the value(s) of the content hash used for this asset
+   */
+  contenthash?: string | Array<string>
+  sourceFilename?: string
```

Comment:
Why the comments for `sourceFilename` is not generated here

Reply:
should use `///`, i will fix it.